### PR TITLE
Include Lombok dependency in POM.xml + Use Lombok annotations.

### PR DESCRIPTION
**Pull request by DzmitryNovik**

### CODE REVIEW

File: src/main/java/com/team4/testingsystem/entities/ChosenOption.java [hunk 1]
```diff
@@ -13,6 +17,11 @@
 
 @Entity
 @Table(name = "chosen_option")
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
```
Comment (AntonMaksimenkau) [inline src/main/java/com/team4/testingsystem/entities/ChosenOption.java:24]: Why don't you like to use `@Data` annotation instead of a list of other ones?

**Review by AntonMaksimenkau — CHANGES_REQUESTED** [review]
Please, also use `@Builder` annotation instead a custom ones

**Comment by AntonMaksimenkau** [comment]
BTW, what about the rest of the project? Are you going to make a separate PR for it?

File: src/test/java/com/team4/testingsystem/converters/TestConverterTest.java [hunk 2]
```diff
@@ -98,6 +99,7 @@ void convertToDTO() {
                 chosenOptions.add(new ChosenOption(new TestQuestionID(test, question), answers.get(0)));
 
                 question.setAnswers(answers);
+                /*questions.add(question);*/
```
Comment (AntonMaksimenkau) [inline src/test/java/com/team4/testingsystem/converters/TestConverterTest.java]: why?

**Review by AntonMaksimenkau — CHANGES_REQUESTED** [review]

File: src/test/java/com/team4/testingsystem/converters/TestConverterTest.java (hunk 2 above)
Comment (DzmitryNovik) [reply src/test/java/com/team4/testingsystem/converters/TestConverterTest.java]: It was an attempt to fix error. This line is not needed.

Comment (AntonMaksimenkau) [reply src/test/java/com/team4/testingsystem/converters/TestConverterTest.java]: I'm pretty sure we need it.
Or do we have another place where we are setting the questions?

**Review by AntonMaksimenkau — APPROVED** [review]

**Review by maxim12321 — APPROVED** [review]

**Pull request merged**